src/main/java/core/ReadProperties.java

```diff
@@ -0,0 +1,30 @@
+package core;
+
+import java.io.IOException;
+import java.util.Properties;
+
+public class ReadProperties {
+
+    private static ReadProperties instance;
+    protected static Properties properties;
+
+    private ReadProperties() {
+        properties = new Properties();
+        try {
+            properties.load(getClass().getClassLoader().getResourceAsStream("config.properties"));
```

Comment:
Please move property file path to a separate constant variable

Reply:
Please use 'try with resources' for loading properties

Reply:
done